examples/toolkit/scripts/wordcloud.py

```
@@ -0,0 +1,102 @@
+    "there's", "here's", "what's", "who's", "let's", "can't", "won't", "shouldn't",
+    "been", "being", "going", "know", "think", "want", "need", "use", "using",
+    "used", "every", "other", "only", "same", "well", "back", "good", "best",
+    "better", "right", "thing", "things", "something", "anything", "nothing",
+    "someone", "anyone", "everyone", "people", "time", "year", "years", "day",
+    "long", "great", "lot", "amp", "http", "https", "www", "com",
+}
+
+
+def fetch_posts(subreddit, sort, count=25):
+    url = f"https://www.reddit.com/r/{subreddit}/{sort}.json?limit={count}&t=week"
+    req = urllib.request.Request(url, headers={"User-Agent": USER_AGENT})
```

Ruff (0.15.5)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)